runhouse/resources/hardware/launcher_utils.py

```diff
@@ -208,7 +208,9 @@ def up(cls, cluster, verbose: bool = True, force: bool = False):
                 payload=payload,
                 cluster_name=payload["cluster_config"].get("name"),
             )
-            cls._update_from_den_response(cluster=cluster, config=data)
+            cls._update_from_den_response(
```

Comment:
actually don't we call this from two places? this addresses the streaming use case, but i imagine the same would apply if verbose=False?

Reply:
if verbose=False, the data dictionary we receive equals to the cluster_config. It is not a dict with two key. Is it expected or it is also might be related to the `is_up` issues?